rtpsender.go

```diff
@@ -437,11 +437,16 @@ func (r *RTPSender) ReadRTCP() ([]rtcp.Packet, interceptor.Attributes, error) {
 func (r *RTPSender) ReadSimulcast(b []byte, rid string) (n int, a interceptor.Attributes, err error) {
 	select {
 	case <-r.sendCalled:
+		r.mu.Lock()
 		for _, t := range r.trackEncodings {
 			if t.track != nil && t.track.RID() == rid {
-				return t.rtcpInterceptor.Read(b, a)
+				reader := t.rtcpInterceptor
```

Comment:
does `Read` also need it? Maybe for consistency?

Reply:
Read will block so can't put it under the lock. It is ok since the same RID encoding has very low chance to change and read a previous rtcp packet does not matter even if it is changed